scripts/mk_nolint.lean

```diff
@@ -0,0 +1,37 @@
+/-
+Copyright (c) 2019 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Robert Y. Lewis
+-/
+
+import tactic.lint system.io data.list.sort all
```

Comment:
The first three are the ones you need for this file to work. The `all` import is so it produces the output we need to use. I thought it might be better to leave the imports like this in case we want to run it on a subset of the library, or comment out `all` for testing purposes. It's not a big deal though and I'll change it if you care.

Reply:
```suggestion
import tactic.lint system.io data.list.sort   -- these are required
import all   -- then import everything, to parse the library for failing linters
```